content/blog/2023/12/kubewarden-2023-wrapped.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Kubewarden 2023 Wrapped
+authors:
+  - Flavio Castelli
+date: 2023-12-20
+---
+
+The end of the year is around the corner. Let's look at what the Kubewarden project achieved in 2024!
+
+## Context-Aware graduation
+
+The context-aware feature graduated to stable during this year. It did that by going through a massive overhaul compared to its initial iteration.
+Context-aware policies can access information about Kubernetes resources defined inside the cluster. At evaluation time, these policies can make decisions based on this information.
+Such an example is the [unique ingress host policy](https://artifacthub.io/packages/kubewarden/unique-ingress-policy/ingress-unique-host).
+
+Policy, by default, has no access to the cluster information. Kuberwarden administrators can grant read-only access to specific resources. The control access operates on a per-policy level, which allows fine-tuned settings for each deployed policy.
+
+This feature is available to all kinds of Kubewarden policies, except for the WASI ones (yet). Context-aware policies written for Open Policy Agent/Gatekeeper do not need any change.
+
+You can learn more about context-aware policies [here](https://docs.kubewarden.io/explanations/context-aware-policies).
+
+## WASI policies
+
+This year, we introduced a new type of policy: the WASI one. This type of policy allows to experiment with bleeding-edge WebAssembly features, which can be attractive for advanced policy authors.
+
+We created WASI policies because we wanted to use the latest changes introduced by Go 1.21.
+For example, we relied on a WASI policy to write [this policy](https://github.com/kubewarden/kyverno-dsl-policy) that allows the reuse of Kyverno policies.
+
+You can learn more about WASI policies [here](https://docs.kubewarden.io/writing-policies/wasi/intro-wasi).
+
+## Audit Scanner
+
+Administrators enforce new policies over time and change the settings of the existing ones. A conformant resource may suddenly become non-compliant because of one of these changes.
```

Comment:
It might be better to use conformant and non-conformant, or compliant and non-compliant. Rather than mixing it up. Compliance is the better word but either will do.

Reply:
fixed